cmd/mesh/testdata/manifest-generate/output/pilot_override_kubernetes.yaml

```diff
@@ -190,6 +190,32 @@ spec:
     istio: pilot
 
 ---
+apiVersion: policy/v1beta1
```

Comment:
This file wasn't modified in https://github.com/istio/operator/pull/286. What PR modified it in `master`?

Reply:
The charts and values in istio/installer are not identical between master and release-1.3, which leads to differences in the golden outputs. This is also why often patches do not apply cleanly and must be manually merged.

Reply:
This was not my question. I need to be able to trace backported changes back to their original `master` PR. So in which `master` PR was this change made?